nva-datamodel-java/src/main/java/no/unit/nva/model/instancetypes/media/AbstractMediaInstance.java

```diff
@@ -0,0 +1,64 @@
+package no.unit.nva.model.instancetypes.media;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
+import no.unit.nva.model.instancetypes.PublicationInstance;
+import no.unit.nva.model.pages.NullPages;
+import nva.commons.core.JacocoGenerated;
+
+import java.util.Objects;
+
+public abstract class AbstractMediaInstance implements PublicationInstance<NullPages> {
+
+    private static final String PAGES = "pages";
+    public static final String CONTRIBUTION_TYPE = "contributionType";
+
+    @JsonProperty(CONTRIBUTION_TYPE)
+    private final ContributionType contributionType;
+    @JsonProperty(PAGES)
+    private static final NullPages pages = new NullPages();
+
+    protected AbstractMediaInstance(@JsonProperty(CONTRIBUTION_TYPE) ContributionType contributionType) {
+        this.contributionType = contributionType;
+    }
+
+    @JsonGetter(CONTRIBUTION_TYPE)
+    public ContributionType getContributionType() {
+        return contributionType;
+    }
+
+    @JsonGetter(PAGES)
+    @Override
+    public NullPages getPages() {
+        return pages;
+    }
+
+    @JsonSetter(PAGES)
+    @Override
+    public abstract void setPages(NullPages pages);
```

Comment:
this should be the exact opposite of abstract. it should be final. The decision of what one can do here has been set by the type of the input parameter